cirq/sim/density_matrix_simulator.py

```diff
@@ -533,19 +532,18 @@ def __init__(self, params: study.ParamResolver,
         self.final_density_matrix = np.reshape(
             final_simulator_state.density_matrix, (size, size))
 
-    def _value_equality_values_(self):
+    def _value_equality_values_(self) -> Any:
         measurements = {
             k: v.tolist() for k, v in sorted(self.measurements.items())
         }
         return (self.params, measurements, self._final_simulator_state)
 
-    def __str__(self):
+    def __str__(self) -> str:
         samples = super().__str__()
-        return 'measurements: {}\nfinal density matrix:\n{}'.format(
-            samples, self.final_density_matrix)
-
-    def __repr__(self):
-        return ("cirq.DensityMatrixTrialResult(params={!r}, measurements={!r}, "
-                "final_simulator_state={!r})".format(
-                    self.params, self.measurements,
-                    self._final_simulator_state))
+        return (f'measurements: {samples}\n'
+                f'final density matrix:\n{self.final_density_matrix}')
+
+    def __repr__(self) -> str:
+        return ("cirq.DensityMatrixTrialResult("
```

Comment:
Should this use single quotes for consistency?

Reply:
Done.